### DISCUSSION TITLE
Next.js 11: @babel/core is not recognized by other packages

### DISCUSSION THREAD

**Discussion by sj-rai** [vercel/next.js#27893] [Help] [answered]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14.16.0

### What browser are you using?

Chrome, Firefox

### What operating system are you using?

macOS

### How are you deploying your application?

yarn build does not work, so not able to deploy.

### Describe the Bug

In next.js 11, babel is not found by other package. I am using a markdown loader, `frontmatter-markdown-loader` for parsing markdown file, which needs `@babel/core` and `@babel/preset-react`, but as I understand, these presets should already be present in `next/babel`. But it is not found, hence `next dev` fails. 

Also, the `next build` fails too, with the error ,`Error: > Build failed because of webpack errors` - which is also due to these babel presets not being found. 

I tried using next.js 10, and it is working fine, there is no issues with `next dev` or `next build`. Also, I don't see `@babel/core` or `@babel/preset-react` in node_modules for next11, but in next10 I see it. Though additionally installing these two presets will fix the problem, but if it is an actual issue it should be fixed in next/babel itself.

### Expected Behavior

`yarn dev` and `yarn build` should run successfully. Any babel presets that next uses, should also be usable by other packages.

### To Reproduce

This is my next.config.js
```
module.exports = {
  reactStrictMode: true,
  webpack: (cfg) => {
    cfg.module.rules.push(
        {
          test: /\.md$/,
          loader: 'frontmatter-markdown-loader',
          options: {mode: ['react-component']}
        }
    )
    return cfg;
    },
}
```
With this config, `next build` fails because `frontmatter-markdown-loader` is not able to find `@babel/core` or `@babel/preset-react` on next.js 11.

**Comment by timneutkens** [comment, marked as answer]
Next.js 11 includes Babel and such in the package itself, previously you were accidentally relying on these dependencies being installed. You'll want to install them as they're dependencies of `frontmatter-markdown-loader`.
